src/frontend/e2e/tests/upload.spec.ts

```diff
@@ -11,19 +11,23 @@ test.describe("Upload sample tests", () => {
   });
 
   fileExtensions.forEach((extenstion) => {
-    test(`Should upload ${extenstion.toUpperCase()} sample file`, async ({
+    test.skip(`Should upload ${extenstion.toUpperCase()} sample file`, async ({
```

Comment:
why are we skipping this now

Reply:
The tests for upload are failing and fixes are in a separate PR. So this will be unskipped in that PR